applications/luci-app-samba4/Makefile

```diff
@@ -5,6 +5,6 @@ include $(TOPDIR)/rules.mk
 LUCI_TITLE:=Network Shares - Samba 4 SMB/CIFS fileserver
 LUCI_DEPENDS:=+samba4-server
 
-include ../../luci.mk
+include $(TOPDIR)/feeds/luci/luci.mk
```

Comment:
This change looks wrong to me.
You change from relative notation (from this package) to absolute path, making also auumptions about the build system.

Reply:
luci-app-omcproxy was doing the same, so i used this style, since it fixes a issue i had if users wanted to build from my external feed. The problem was i had to maintain two versions of my luci app, since the relative notation did not work from other feeds, so i switched to this.
So far it did work for me, but if this is a issue i can change it back, its just inconvenient to have to remember to change the path if i update my apps for my feed + luci repo.

Reply:
ok before this gets stuck any longer i just reverted this change.